lib/kitchen_sink/boolean.ex

```diff
@@ -0,0 +1,25 @@
+defmodule KitchenSink.Boolean do
+
+  @moduledoc """
+  this module is for Boolean functions
+  """
+
+  @doc """
+  Parse boolean either from string or integer, if it's not a boolean then it returns :error
+  """
+  def parse(st) when is_binary(st) do
+    String.downcase(st)
+    |> case do
+      "true" -> true
+      "false" -> false
```

Comment:
We are also using parse since we are using that for Integer and Float. If we want to_boolean then we have to throw an exception like the others.